app/admin/lessons.rb

```diff
@@ -56,11 +56,7 @@
       lesson.lesson_files.map do |file|
         next if file.nil? || !file.blob.filename.instance_variable_get("@filename").present?
         if current_user.admin_write?
-          if file.blob.content_type == "text/plain"
-            link_to file.blob.filename.instance_variable_get("@filename"), file.blob.filename.instance_variable_get("@filename"), target: '_blank'
-          else
-            link_to "#{file.blob.filename}", "#{Rails.application.routes.url_helpers.rails_blob_path(file, only_path: true)}", target: '_blank'
-          end
+          link_to "#{file.blob.filename}", "#{Rails.application.routes.url_helpers.rails_blob_path(file, only_path: true)}", target: '_blank'
```

Comment:
Just to confirm, 
tasks issues and risks, They all have same implementation. We are saving links in plain text content type
https://github.com/MicroHealthLLC/mPATH/blob/master/app/models/lesson.rb#L37